20
20
Sequences folder exists! Using sequences from /home/zht/Datasets/Semantic
parsing seq 00
parsing seq 01
parsing seq 02
parsing seq 03
parsing seq 04
parsing seq 05
parsing seq 06
parsing seq 07
Using 16338 scans from sequences [0, 1, 2, 3, 4, 5, 6, 7]
20
Sequences folder exists! Using sequences from /home/zht/Datasets/Semantic
parsing seq 08
Using 4070 scans from sequences [8]
Loss weights from content:  tensor([  0.0000,  22.9317, 857.5627, 715.1100, 315.9618, 356.2452, 747.6170,
        887.2239, 963.8915,   5.0051,  63.6247,   6.9002, 203.8796,   7.4802,
         13.6315,   3.7339, 142.1462,  12.6355, 259.3699, 618.9667])
Traceback (most recent call last):
  File "train.py", line 118, in <module>
    trainer = Trainer(ARCH, DATA, FLAGS.dataset, FLAGS.log, FLAGS.pretrained)
  File "/home/zht/github_play/SqueezeSegV3/src/tasks/semantic/modules/trainer.py", line 91, in __init__
    self.path)
  File "/home/zht/github_play/SqueezeSegV3/src/tasks/semantic/modules/segmentator.py", line 22, in __init__
    self.ARCH["backbone"]["name"] + '.py')
AttributeError: module '__init__' has no attribute 'TRAIN_PATH'